sources/packages/backend/apps/api/src/constants/error-code.constants.ts

```diff
@@ -150,3 +150,17 @@ export const APPLICATION_WITHDRAWAL_TEXT_CONTENT_FORMAT_ERROR =
  */
 export const APPLICATION_WITHDRAWAL_INVALID_TEXT_FILE_ERROR =
   "APPLICATION_WITHDRAWAL_INVALID_TEXT_FILE_ERROR";
+/**
+ * An error was found during the application bulk withdrawal validation and the process
+ * must be stopped. Every application bulk withdrawal validation can generate an error but
+ * some errors are classified as warnings and therefore are not considered
+ * critical, which means that critical errors are the only ones that will
+ * cause the process to be interrupted.
+ */
+export const APPLICATION_BULK_WITHDRAWAL_VALIDATION_CRITICAL_ERROR =
```

Comment:
We have the text file format errors with error name `APPLICATION_WITHDRAWAL_TEXT_CONTENT_FORMAT_ERROR` isn't that going to fall under this same category of critical error?